### ISSUE TITLE
Duolingo API

### ISSUE THREAD

**Issue by Mardoqueu** [tschuy/duolingo#1]
Is this supposed to build the entire duolingo web?

**Comment by tschuy** [comment]
No, this is an (entirely unofficial) python library. So far it's sort of languished a bit, and I'm not sure if I'm going to get back to using it.
